server/src/main/resources/db/changelog/20170421055601-add-recipient-owner-to-consumer.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="20170421055601-1" author="vrjain">
+        <comment> add-recipient-owner-to-consumer</comment>
+        <addColumn tableName="cp_consumer">
+            <column name="recipient_owner_key" type="VARCHAR(255)"/>
+        </addColumn>
+        <rollback>
+            <dropColumn tableName="cp_consumer" columnName="recipient_owner_key" />
```

Comment:
You [don't need](http://www.liquibase.org/documentation/rollback.html) a custom rollback for `addColumn`.  Liquibase handles it automatically since it's a very basic change.  See the "How Rollback SQL is controlled" section in the link I provided.